apps/dapp-console/app/faucet/components/Faqs.tsx

```diff
@@ -59,7 +59,8 @@ const accordionItems = [
         </Text>
         <Text as="p">
           <b>Gitcoin Passport</b> is an identity verification application. It
-          helps you collect “stamps” that prove your humanity and reputation.
+          helps you collect “stamps” that prove your humanity and reputation. Please note, 
+          the faucet requires a passport score > 25.
```

Comment:
You need to escape the `>` for tests to pass putting `{'>'}` instead of `>` should do the trick